lib/portlayer/exec/commit.go

```diff
@@ -35,6 +35,13 @@ func Commit(ctx context.Context, sess *session.Session, h *Handle, waitTime *int
 	defer trace.End(trace.Begin(h.ExecConfig.ID))
 
 	c := Containers.Container(h.ExecConfig.ID)
+
+	// check for an in memory container migration and reject the commit if it is the case.
+	if c.ExecConfig.Migrated {
```

Comment:
check the migration status at here:
https://github.com/vmware/vic/pull/4045/commits/d5c7717969793557f357f1ba969d17bf14669755#diff-8207405211ed1d9756b8e63d8396f253R106